tests/js/lucene-tests.js

```diff
@@ -1,5 +1,9 @@
 // Sanity checks for search integration.  As these are read only, we use a single instance for all tests.
 "use strict";
+
+var events = require("events");
+events.EventEmitter.defaultMaxListeners = 100;
```

Comment:
Need a comment tracing what problem this works around - and if possible with an issue link for whatever upstream project the problem is with